### ISSUE TITLE
ARM build for Linux

### ISSUE THREAD

**Issue by sorry-i-am-late** [nextcloud/talk-desktop#958]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

I suggest an ARM build for linux as this would help people like the raspberry pi and pine64 community greatly.

Using the browser is technically an alternative but it is far from fun on certain devices.

ARM machines are often lower resource and more battery dependent so they are in even more need of prebuilt releases since compiling from source on something like a pinephone is far from ideal.

**Comment by nickvergessen** [comment]
- Related #871

**Comment by Fuseteam** [comment]
even a zip release would be appreciated

**Comment by ShGKme** [comment]
I had a look at it on the weekend.

Distributions that I'd target and test on:
- Classics: Ubuntu, Fedora, Debian
- Raspberry PI OS
- Maybe builds from pine64, but likely covered by options above

I'd not consider Ubuntu Touch and similar distributions as the target. They are targeting mobile area rather than the desktop.

---

Flatpak release with Flathub may support ARM64 build easily.
Other distributions may be problematic, but not impossible.

**Comment by Fuseteam** [comment]
for completeness the snapstore is also capable of doing arm64 builds

as for Ubuntu touch and similar, as long as we have an arm binary, the community could experiment with it— at best the interface of talk would need to be responsive..... would you be open to PRs to that effect?

**Comment by ShGKme** [comment]
> would you be open to PRs to that effect?

For what specifically?

**Comment by Fuseteam** [comment]
> > would you be open to PRs to that effect?
> 
> For what specifically?

for adjusting the UI based on screen size for example— not that i have a proof of concept at the moment

**Comment by sorry-i-am-late** [comment]
May I suggest packaging for alpine if possible due to postmarketOS having a
very wide range of ARM devices and flathub for versitilities sake.

**Comment by ShGKme** [comment]
postmarketOS seems to target mobile platforms, not desktop

**Comment by ShGKme** [comment]
> for adjusting the UI based on screen size for example

We have it

**Comment by Fuseteam** [comment]
> > for adjusting the UI based on screen size for example
> 
> We have it

yeah i think we'll just have to test once we have arm64 binaries, to see what the actual hurdles are 😂

**Comment by sorry-i-am-late** [comment]
PostmarketOS also has good support for things like ARM chromebooks and
other tablet to laptop class items.

**Comment by keithzg** [comment]
Yeah, while I frankly was considering settling for this app for even some handhelds due to the lack of any mobile-friendly UI that I could find for Talk outside of proprietary OSes, it also vexes me not to have a "native" (though Electron is pretty heavy) app on my PineBookPro running postmarketOS either, or the Lenovo Chromebook that I likewise installed postmarketOS on due to great ARM support in general and for those devices in particular.
